packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -407,6 +408,15 @@ class ExpansionTile extends StatefulWidget {
   ///   [ExpansionTileThemeData].
   final EdgeInsetsGeometry? childrenPadding;
 
+  /// The color to display behind the expanded children section of the tile.
+  ///
+  /// If this property is null, the value from [ExpansionTileThemeData.childrenBackgroundColor] is
+  /// used if specified. If both are null, then [backgroundColor] will be used.
+  ///
+  /// * [ExpansionTileTheme.of], which returns the nearest [ExpansionTileTheme]'s
```

Comment:
This explanation looks a little extra since it doesn't have a strong relationship with this property:) So I think we can remove this:)